packages/react-ui-components/src/components/ToasterController.types.ts

```diff
@@ -0,0 +1,14 @@
+import { EToasterType } from '@kelvininc/ui-components';
+
+export interface IToasterState {
```

Comment:
Should this interface extend the interface `IToaster` from Toaster?
Or have separated states, if we use functional components. one to show and another with the toaster info. @djcouto help us here

Reply:
I agree with the change.